child_compassion/views/global_childpool_view.xml

```diff
@@ -130,6 +128,31 @@
             </field>
         </record>
 
+        <record id="view_compassion_global_child_attributes_form" model="ir.ui.view">
+            <field name="name">child.search.attributes.form</field>
+            <field name="model">child.search.attributes</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet>
+                        <group>
+                            <field name="name"/>
+                            <field name="type"/>
+                            <field name="hold_expiration_date"/>
+                            <field name="primary_owner"/>
+                            <field name="secondary_owner"/>
+                            <field name="no_money_yield_rate"/>
+                            <field name="yield_rate"/>
+                        </group>
+                        <footer>
+                            <button name="send" string="Validate" type="object" class="oe_highlight"/>
+                            or
```

Comment:
New guidelines for odoo views is to avoid cancel button, because the user can simply close the popup and that acts as a cancel. There is no need to add a button for that.